src/components/common/Certify.vue

```diff
@@ -38,7 +38,7 @@
           </template>
         </v-checkbox>
         <p class="certify-clause signature-date">
-          <strong>Date:</strong> {{ formattedCurrentDate }}
+          <strong>Date:</strong> {{ formattedCurrentDate || '[unknown]' }}
```

Comment:
Any particular reason why "[unknown]" is lowercase here but is capitalized as "[Unknown]" in LimitedRestorationExtensionFiling.vue?

Reply:
I'll update that other component.

We haven't had much UX guidance on error handling, but I've gravitated towards showing unknown as `[unknown]` (which looks like a token) instead of `Unknown` (which looks like a noun). Let me know your thoughts on that.

Reply:
I'm personally good with either if it's consistent!

Reply:
It is now!